apps/cowswap-frontend/src/modules/trade/containers/TradeWidget/TradeWidgetForm.tsx

```diff
@@ -39,7 +39,8 @@ import { WrapFlowActionButton } from '../WrapFlowActionButton'
 const scrollToMyOrders = () => {
   const element = document.getElementById(MY_ORDERS_ID)
   if (element) {
-    element.scrollIntoView({ behavior: 'smooth' })
+    const elementTop = element.getBoundingClientRect().top + window.scrollY - 76
```

Comment:
Need comments here, what 76 does mean?

Reply:
Addressed this. This is the sticky header offset. Made more obvious by moving to const with a comment.